itez/beneficiary/tests/test_utils.py

```diff
@@ -0,0 +1,39 @@
+from django.test import TestCase
+
+from itez.beneficiary.utils import generate_uuid_and_agent_code
+
+
+class UUIDAndAgentCode(TestCase):
+    @classmethod
+    def setUp(self):
+        """
+         This method is called for every test that runs and helps with variable declaration
+        """
+        self.uuid_code = generate_uuid_and_agent_code()[0]
+        self.agent_code = generate_uuid_and_agent_code()[1]
+
+        return super().setUp(self)
```

Comment:
This is just a set up why are we returning this
Are you referencing the Django docs on tests or you are just writing them

Reply:
- [x] remove return statements